h/api/models/document.py

```diff
@@ -3,15 +3,18 @@
 from __future__ import unicode_literals
 
 from datetime import datetime
+import logging
 
 import sqlalchemy as sa
 from sqlalchemy.dialects import postgresql as pg
 from sqlalchemy.ext.hybrid import hybrid_property
-from sqlalchemy.orm import aliased
 
-from h.api import uri
 from h.api.db import Base
 from h.api.db import mixins
+from h.api.uri import normalize as uri_normalize
```

Comment:
There's an annoying problem here. `create_or_update_document_uri()` has a parameter called `uri` which shadows the `uri` imported here (if we did `from h.api import uri`) and means that `create_or_update_document_uri()` can't use the `uri` module.

I would just rename the parameter to `uri_` but I want `storage` to be able to do `create_or_update_document_uri(**data)` with the validated data from the client. And the field that the client sends is called `uri` not `uri_`.

So I changed this to import the function directly which is a bit naughty and aliased the function to make it clearer.